app/views/hosts/_form.html.erb

```diff
@@ -55,18 +55,18 @@
          be different from the defaults, or the defaults could have changed after the vm was created. %>
         <div id="compute_profile" <%= display?(!@host.compute_resource_id) %> >
           <%= select_f f, :compute_profile_id, ComputeProfile.visibles, :id, :name,
-            { :include_blank => true },
+            { :disable_button => _("inherit") },
```

Comment:
would it make sense to reuse `HostsAndHostgroupsHelper::OVERRIDE_TEXT` here and below?